base/abstractarraymath.jl

```diff
@@ -92,6 +92,10 @@ imag{T<:Real}(x::AbstractArray{T}) = zero(x)
 +{T<:Number}(x::AbstractArray{T}) = x
 *{T<:Number}(x::AbstractArray{T,2}) = x
 
+function lerpi(j::Integer, d::Integer, A::AbstractArray, B::AbstractArray)
```

Comment:
I can test. But curiously I'm getting a 2x performance improvement by writing out `lerpi` explicitly like this:
```jl
function lerpi(j::Integer, d::Integer, A::AbstractArray, B::AbstractArray)
    out = similar(A)
    @inbounds for I in eachindex(A)
        out[I] = lerpi(j, d, A[I], B[I])
    end
    out
#    lerpi.(j, d, A, B)
end
```
While it's nice to get better performance, it's sort of backward progress from the standpoint of code prettiness.